include/Reader/readerir.h

```diff
@@ -1453,6 +1524,15 @@ class GenIR : public ReaderBase {
   std::vector<llvm::Value *> Arguments;
   llvm::Value *IndirectResult;
   llvm::DenseMap<uint32_t, llvm::StoreInst *> ContinuationStoreMap;
+  llvm::SmallPtrSet<llvm::Value *, 5> StructPointers; ///< This set contains
+                                                      ///< pointers to structs
+                                                      ///< that we put on
+                                                      ///< the operand stack
+                                                      ///< to represent structs
+                                                      ///< since we don't allow
+                                                      ///< first-class
+                                                      ///< aggregates in LLVM IR
+                                                      ///< we are generating.
```

Comment:
I'll update the comment. 5 is an arbitrary small int. I think in the common case we won't have many of these in a method. Do you have a suggestion for a different const or a different data structure here?